ios/Helio.xcodeproj/xcshareddata/xcschemes/HelioStaging.xcscheme

```diff
@@ -15,9 +15,9 @@
                   <BuildableReference
                      BuildableIdentifier = "primary"
                      BlueprintIdentifier = "888DAA1429B48C8900311304"
-                     BuildableName = "BetterSocialStaging.app"
-                     BlueprintName = "BetterSocialStaging"
-                     ReferencedContainer = "container:BetterSocial.xcodeproj">
+                     BuildableName = "HelioStaging.app"
+                     BlueprintName = "HelioStaging"
+                     ReferencedContainer = "container:Helio.xcodeproj">
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The app name and container reference have been changed from "BetterSocial" to "Helio". Make sure that all references to the old name have been updated throughout the project, including any scripts or documentation that may refer to the old name.
```diff
-   BuildableName = "BetterSocialStaging.app"
-   BlueprintName = "BetterSocialStaging"
-   ReferencedContainer = "container:BetterSocial.xcodeproj"
+   BuildableName = "HelioStaging.app"
+   BlueprintName = "HelioStaging"
+   ReferencedContainer = "container:Helio.xcodeproj"
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->